packages/web-components/src/components/footer/language-selector.ts

```diff
@@ -0,0 +1,138 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { html, property, customElement, query } from 'lit-element';
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings.js';
+import BXComboBox from 'carbon-web-components/es/components/combo-box/combo-box.js';
+import BXComboBoxItem from 'carbon-web-components/es/components/combo-box/combo-box-item.js';
+import { FOOTER_SIZE } from './footer';
+import styles from './footer.scss';
+import { findIndex, forEach } from '../../globals/internal/collection-helpers';
+
+const { stablePrefix: ddsPrefix } = ddsSettings;
+
+/**
+ * Language selector
+ *
+ * @element dds-language-selector
+ */
+@customElement(`${ddsPrefix}-language-selector`)
+class DDSLanguageSelector extends BXComboBox {
+  @property()
+  lastValidLang = 'English';
+
+  /**
+   * Size property to apply different styles.
+   */
+  @property()
```

Comment:
Attribute names should be dash-cased:

```suggestion
  @property({ attribute: 'footer-size' })
```